content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
```

Comment:
I'm keeping this, since we don't want to remove stuff from the current rule book. But it shouldn't belong long-term, since it's subjective, and shouldn't (can't) be the grounds for challenge/protest.

Reply:
I'm OK with leaving it in, but I also think removing it doesn't move us away from func equiv. The reason: This sentence doesn't mean anything. It's so subjective as to be pointless. I wouldn't even include this sentence in a best practices doc unless it was followed with clear examples. That said, let's have Jeremy or Zachary weigh before doing anything.

Reply:
Ah, the irony of "clear and precise" being a vague and imprecise rule. I agree with Gryphon.

Reply:
you agree with Gryphon in removing it?

Reply:
Or do you agree with Gryphon in leaving it in? ;)

Reply:
The irony of a vague and imprecise comment on a vague and imprecise "clear and precise" rule. :)

Reply:
I'd be fine either way. I might lean towards keeping it in for now just to prevent people from saying "but that line is completely missing!" Though I do think that one can be easily explained.

Reply:
cool, I'm removing it.